torch/_inductor/triton_heuristics.py

```diff
@@ -153,6 +153,7 @@ def __init__(
         self.mutated_arg_names = mutated_arg_names
         self.configs = configs
         self.heuristic_type = heuristic_type
+        self.device_interface = get_interface_for_device(self.triton_meta["device_type"])
```

Comment:
How about renaming it as `gpgpu_device`? Current device abstraction is modeling the gpgpu device for computation. Therefore, I think it is better to make it more specific.